### PR TITLE
Update release rate for etcd

### PR DESCRIPTION
**Pull request by ahrtr**
Follow up https://github.com/cncf/landscape/pull/3123#discussion_r1160377986 

cc @jmhbnz @mitake @ptabor @serathius @spzala

### CODE REVIEW

**Review by spzala — COMMENTED** [review]
lgtm 
Thanks @ahrtr

**Review by jmhbnz — APPROVED** [review]
LGTM

**Comment by serathius** [comment]
LGTM

File: landscape.yml [hunk 1]
```diff
@@ -3057,18 +3057,18 @@ landscape:
...
                 By using etcd, developers can ensure that their applications have access to up-to-date configuration data, even as they scale up or down, and
                 can maintain consistency, fault tolerance and coordination across multiple instances of the application.
-              summary_business_use_case: >
+              summary_business_use_case: >-
+                Etcd can be used by businesses to manage the configuration data and coordination needs of their applications, ensuring consistent and reliable performance at scale.
+              summary_release_rate: Major or Minor release approximately every 2 years, and patch release every 1-3 months
+              summary_integrations: Due to etcd's flexible and extensible architecture it has been integrated as a data store for many projects such as Kubernetes, CoreDNS and Hashicorp Vault.
                 Etcd can be used by businesses to manage the configuration data and coordination needs of their applications, ensuring consistent and reliable
                 performance at scale.
               summary_release_rate: Every 1-3 months
```
Comment (ptabor) [inline landscape.yml:3071]: This seems redundant now.

Comment (ahrtr) [reply landscape.yml:3071]: I removed it in my first commit https://github.com/cncf/landscape/pull/3128/commits/5f9283d26f17921a5ed30f6f093511facd465077, but it was added back in the second & third commits.

**Comment by ahrtr** [comment]
I just squashed the commits, and resolved all conflicts.

**Comment by serathius** [comment]
LGTM

**Pull request merged**